android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -610,7 +611,7 @@ private void removeView(View view, @Nullable ViewGroup parent) {
       mReanimatedNativeHierarchyManager.publicDropView(view);
     }
 
-    if (parent != null) {
+    if (parent != null && tag != -1) {
```

Comment:
If I understand correctly, `view` is a child of `parent`. Could you explain me why we can't remove from his parent?